cookbooks/cdo-varnish/test/shared/shared.rb

```diff
@@ -206,11 +206,12 @@ def build_url(prefix='x', suffix='')
   end
 
   it 'Handles Accept-Language behaviors' do
-    url = build_url 3
+    # URL contains whitelisted 'Accept-Language' header
+    url = build_url 's/starwars/stage/1/puzzle'
     text_en = 'Hello World!'
     text_fr = 'Bonjour le Monde!'
-    mock_response url, text_en, {'X-Varnish-Accept-Language' => 'en'}, {vary: 'X-Varnish-Accept-Language'}
-    mock_response url, text_fr, {'X-Varnish-Accept-Language' => 'fr'}, {vary: 'X-Varnish-Accept-Language'}
+    mock_response url, text_en, {'X-Varnish-Accept-Language' => 'en'}
+    mock_response url, text_fr, {'X-Varnish-Accept-Language' => 'fr'}
```

Comment:
Note that the explicit `vary` header can now be removed from the mock responses in this `Accept-Language` integration test, because the same functionality is now automatically handled by the header whitelist in Varnish (as it already was in CloudFront).